ReleaseProcessAutomation/Configuration/Data/Settings.cs

```diff
@@ -24,6 +24,8 @@ namespace ReleaseProcessAutomation.Configuration.Data;
 
 public class Settings
 {
+  public string ConfirmedMSBuildPath { get; set; }
```

Comment:
Actually I think it would be better two have both versions! Have MsBuildPath as standard and add a secondary (like alternativeMsBuildPath) to the config, so we dont have unnecessary breaking changes

Reply:
We can keep the `msBuildPath` naming for now to not break existing configs but I would still rename the property.